routes/runtime/github.rb

```diff
@@ -113,7 +118,12 @@ class CloverRuntime
         fail CloverError.new(400, "InvalidRequest", "Wrong parameters") if etags.nil? || etags.empty? || upload_id.nil? || size == 0
 
         entry = GithubCacheEntry[repository_id: repository.id, upload_id: upload_id, committed_at: nil]
-        fail CloverError.new(204, "NotFound", "No cache entry") if entry.nil? || entry.size != size
+        fail CloverError.new(204, "NotFound", "No cache entry") if entry.nil?
```

Comment:
Should we continue to check size if it's provided?
```suggestion
        fail CloverError.new(204, "NotFound", "No cache entry") if entry.nil? || (entry.size && entry.size != size)
```